Detections/OfficeActivity/MailItemsAccessedTimeSeries.yaml

```diff
@@ -46,9 +46,10 @@ query: |
   // Join against base logs since specified timeframe to retrive records associated with the hour of anomoly
   | join (
       OfficeActivity
-  | where TimeGenerated > ago(2d)
-  | where OfficeWorkload=~ "Exchange" and Operation =~ "MailItemsAccessed" and ResultStatus =~ "Succeeded"
-  ) on TimeGenerated
+      | where TimeGenerated > ago(2d)
+      | extend DateHour = bin(TimeGenerated, 1h)
+      | where OfficeWorkload=~ "Exchange" and Operation =~ "MailItemsAccessed" and ResultStatus =~ "Succeeded"
+  ) on $left.TimeGenerated == $right.DateHour
 entityMappings:
```

Comment:
Think we should include the legacy entity mapping items as those are still needed and IPCustomEntity is not mapped to anything so we need that at a minimum - | extend timestamp = TimeGenerated, AccountCustomEntity = UserId, IPCustomEntity = ClientIP.  Also, ClientIP includes the Port in the field, you will want to remove that so entity mapping can occur properly.